### PR TITLE
Fixed rbg percentage parsing in nsvg__parseColorRGB

### PR DESCRIPTION
**Pull request by AuthorityFX**

### CODE REVIEW

**Review by alerque — APPROVED** [review]

**Comment by oehhar** [comment]
Dear AuthorityFX,
thank you for the code, this is appreciated.
Your proposal is to allow float numbers in the RGB field instead of integers.
Do you have any rational for this?
Remark that a rationale may help the pull request to be merged.
Take care,
Harald

**Comment by AuthorityFX** [comment]
> Dear AuthorityFX, thank you for the code, this is appreciated. Your proposal is to allow float numbers in the RGB field instead of integers. Do you have any rational for this? Remark that a rationale may help the pull request to be merged. Take care, Harald

https://lists.w3.org/Archives/Public/www-svg/2014Jan/0109.html

I'm using assets exported from Adobe InDesign.

Adobe InDesign exports rgb percentage as floats. SVGs from InDesign are handled correctly in other parsers.

File: src/nanosvg.h [hunk 1]
```diff
@@ -1228,8 +1228,9 @@ static unsigned int nsvg__parseColorRGB(const char* str)
 	unsigned int r=0, g=0, b=0;
 	if (sscanf(str, "rgb(%u, %u, %u)", &r, &g, &b) == 3)		// decimal integers
 		return NSVG_RGB(r, g, b);
-	if (sscanf(str, "rgb(%u%%, %u%%, %u%%)", &r, &g, &b) == 3)	// decimal integer percentage
-		return NSVG_RGB(r*255/100, g*255/100, b*255/100);
+	float rf=0, gf=0, bf=0;
```
Comment (memononen) [inline src/nanosvg.h:1231]: definitions go to the top of the block (line after the r,g,b)

**Review by memononen — CHANGES_REQUESTED** [review]

**Comment by memononen** [comment]
@AuthorityFX Thanks for the fix, and thanks @oehhar for triage.

**Comment by oehhar** [comment]
Thank you all for the action, I appreciate !

**Pull request merged**